packages/data/src/redux-store/index.js

```diff
@@ -608,7 +624,7 @@ function mapResolvers( resolvers, selectors, store, resolversCache ) {
  * @param {Array}  args         Selector Arguments.
  */
 async function fulfillResolver( store, resolvers, selectorName, ...args ) {
-	const resolver = get( resolvers, [ selectorName ] );
+	const resolver = resolvers[ selectorName ];
```

Comment:
I hope that `resolvers` can only be an object based on the existing documentation. The only difference with `get` is that would gracefully handle different param types here.

Reply:
It will always be an object. If you trace it to its only usage, you'll notice that [it's called](https://github.com/WordPress/gutenberg/blob/ac2e68f9b7ff291abdc9f2709c7f5943659be4e1/packages/data/src/redux-store/index.js#L567) on the [result of a `mapValues()`](https://github.com/WordPress/gutenberg/blob/ac2e68f9b7ff291abdc9f2709c7f5943659be4e1/packages/data/src/redux-store/index.js#L515-L524), and that will generally only be an object.